packages/gatsby-theme-newrelic/src/hooks/useLoggedIn.js

```diff
@@ -55,8 +55,6 @@ export const useLoggedIn = () => {
   return { loading, loggedIn };
 };
 
-export default useLoggedIn;
```

Comment:
i missed this the first time but `gatsby-theme-newrelic/index.js` reexports the default export from this file as `useLoggedIn`. i pushed `71e5a45` to fix this